lib/trie/child_storage.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // ChildStorageKeyPrefix is the prefix for all child storage keys
-var ChildStorageKeyPrefix = []byte(":child_storage:")
+var ChildStorageKeyPrefix = []byte(":child_storage:default:")
```

Comment:
where is this change from?

Reply:
In the spec, F.1.9 it says this key must be prefixed with `:child_storage:default:`